docs/rest_api_guide/03_RESTservices.rst

```diff
@@ -0,0 +1,393 @@
+            },
+            {
+              "count": 260379,
+              "asset": "lathe1004Vibration"
+            }
+          ],
+          "plugin": {
+            "name": "lathesim",
+            "version": "1.9.2"
+          },
+          "schedule_enabled": true
+        },
+        {
+          "name": "sine2",
+          "address": "localhost",
+          "management_port": 38679,
+          "service_port": 33433,
+          "protocol": "http",
+          "status": "running",
+          "assets": [
+            {
+              "count": 734,
+              "asset": "sine2"
+            },
+            {
+              "count": 373008,
+              "asset": "sine250"
+            }
+          ],
+          "plugin": {
+            "name": "sinusoid",
+            "version": "1.9.2"
+          },
+          "schedule_enabled": true
+        },
+        {
+          "name": "test1",
+          "address": "",
+          "management_port": "",
+          "service_port": "",
+          "protocol": "",
+          "status": "",
+          "assets": [
+            {
+              "count": 76892,
+              "asset": "sinusoid"
+            },
+            {
+              "count": 125681,
+              "asset": "sinusoid2"
+            }
+          ],
+          "plugin": {
+            "name": "sinusoid",
+            "version": "1.9.2"
+          },
+          "schedule_enabled": false
+        },
+        {
+          "name": "testacl",
+          "address": "",
+          "management_port": "",
+          "service_port": "",
+          "protocol": "",
+          "status": "",
+          "assets": [
+            {
+              "count": 76892,
+              "asset": "sinusoid"
+            }
+          ],
+          "plugin": {
+            "name": "testing",
+            "version": "1.9.2"
+          },
+          "schedule_enabled": false
+        },
+        {
+          "name": "dsds",
+          "address": "",
+          "management_port": "",
+          "service_port": "",
+          "protocol": "",
+          "status": "",
+          "assets": [],
+          "plugin": {
+            "name": "Expression",
+            "version": "1.9.2"
+          },
+          "schedule_enabled": false
+        }
+      ]
+    }
+    $
+
+Service Types
+-------------
+
+Fledge supports a number of different service types, some of which are included with the base Fledge installation and others that must be installed separately if required.
+
+.. note::
+
+   The API entry points in this section require that the Fledge installation has been configured with access to a Fledge package repository.
+
+Installed Service Types
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to find out what service types are installed in the system the */fledge/service/installed* call can be used.
+
+.. code-block:: console
+
+    $ curl http://localhost:8081/fledge/service/installed
+    {"services": ["storage", "north", "dispatcher", "notification", "south"]}
```

Comment:
We may add as note for the default/inbuilt services installed are `storage, south, north`